src/components/modals/AddPetModal.tsx

```diff
@@ -24,7 +26,7 @@ export const AddPetModal = () => {
             Add a new pet. Fields marked with '*' are required.
           </DialogDescription>
         </DialogHeader>
-        <AddPetForm />
+        <AddPetForm onSuccess={() => setOpen(false)} />
```

Comment:
wonderful! :D I had thought adding setOpen(true) would be redundant (because it's already in this Modal component), but now the modal closes on submit! Awesome, nicely done! :D